std/conv.d

```diff
@@ -289,6 +289,31 @@ template to(T)
     assert(to!int(to!float(-a)) == -a);
 }
 
+/**
+ * Conversion from string types to char types enforces the input
+ * to have a single code point, and then it converts said code point
+ * to its representation in a single code unit of the target type.
+ * If the code point cannot be represented in a single code unit of
+ * the target type, ConvException is thrown.
+ */
```

Comment:
FYI: we recently added to the [DStyle](https://dlang.org/dstyle.html) that new code [shouldn't not have leading stars](https://dlang.org/dstyle.html#phobos_documentation) - though you are the first one to submit a PR with new documentation since this has been added.

Reply:
Noted and modified, thanks.